### PR TITLE
Add support for Xiaomi power plug ZNCZ03LM

### PR DESCRIPTION
**Pull request by jsliang**
Add support for Xiaomi power plug ZNCZ03LM

### CODE REVIEW

File: devices.js [hunk 1]
```diff
@@ -381,6 +381,20 @@ const devices = [
         ],
         toZigbee: [tz.on_off],
     },
+    {
+        zigbeeModel: ['lumi.plug.mitw01'],
+        model: 'ZNCZ03LM',
+        description: 'Mi power plug ZigBee TW',
```
Comment (Koenkk) [inline devices.js]: What does *TW* mean?

Comment (jsliang) [reply devices.js]: It means that this is the Taiwan (TW) version of Mi power plug ZigBee. AFAIK this model is Taiwan only (with a type C socket & plug).

Comment (Koenkk) [reply devices.js]: ok, thanks!

**Comment by Koenkk** [comment]
Thanks :bee:

**Pull request merged**